React/Views/RCTMapManager.m

```diff
@@ -41,6 +41,7 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(maxDelta, CGFloat)
 RCT_EXPORT_VIEW_PROPERTY(minDelta, CGFloat)
 RCT_EXPORT_VIEW_PROPERTY(legalLabelInsets, UIEdgeInsets)
+RCT_REMAP_VIEW_PROPERTY(mapType, mapType, MKMapType)
```

Comment:
You can leave this at `RCT_EXPORT_VIEW_PROPERTY(mapType, MKMapType)` as the implementation of `RCT_EXPORT_VIEW_PROPERTY` is anyways like this:

```objective-c
#define RCT_EXPORT_VIEW_PROPERTY(name, type) RCT_REMAP_VIEW_PROPERTY(name, name, type)
```